src/backend/api/meals.js

```diff
@@ -1,15 +1,83 @@
-const express = require("express");
-const router = express.Router();
-const knex = require("../database");
+ const express = require("express");
+ const router = express.Router();
+ const knex = require("../database");
 
-router.get("/", async (request, response) => {
+ router.get("/", async (request, response) => {
+   try {
+    // knex syntax for selecting things. Look up the documentation for knex for further info
+     const meals = await knex("meal").select("*");
+     if(meals.length === 0){
+      response.status(404).send("No meals available !")
+     }
+      response.json(meals);
+    } catch (error) {
+      throw response.status(404).send(error);
+    }
+ });
+// POST /api/meals
+
+router.put("/:id", async (request, response) => {
```

Comment:
The path should be `/` for adding a meal as the id is created after successfully adding your meal.